index.html

```diff
@@ -13,35 +13,189 @@
+                <div class="content">
+                    <h2 class="title">Data Dashboard Healthcare</h2>
+                    <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                    <div class="btn">
+                        <button type="button" class="grey-button">HTML</button>
+                        <button type="button" class="grey-button">Bootstrap</button>
+                        <button type="button" class="grey-button">Ruby</button>
+                    </div>
+                        <button type="button" class="orange-button">See project</button>
+                    </div>
+                    
+                </li>
+                <li class="grid-item white"> 
+                    <div class="content">
+                        <h2 class="title">Data Dashboard Healthcare</h2>
+                        <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                        <div class="btn">
+                            <button type="button" class="grey-button">HTML</button>
+                            <button type="button" class="grey-button">Bootstrap</button>
+                            <button type="button" class="grey-button">Ruby</button>
+                        </div>
+                            <button type="button" class="orange-button">See project</button>
+                        </div>
+                        
+                    </li>
+                    <li class="grid-item white"> 
+                        <div class="content">
+                            <h2 class="title">Data Dashboard Healthcare</h2>
+                            <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                            <div class="btn">
+                                <button type="button" class="grey-button">HTML</button>
+                                <button type="button" class="grey-button">Bootstrap</button>
+                                <button type="button" class="grey-button">Ruby</button>
+                            </div>
+                                <button type="button" class="orange-button">See project</button>
+                            </div>
+                            
+                        </li>
+
+    </ul>
+
+    </main>
+
+    <!-- About Section -->
+    <section id="about">
+        <div id="intro">
+        <h2 id="about-me" class="orange-color">About me</h2>
+        <p class="text">Hey there! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+        <button type="button" id="abt-btn"  class="orange-button">Get my resume</button>
```

Comment:
- [ ] Kindly apply button interactions here as well, as I mentioned earlier. Kindly apply this to all buttons in your project.